documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -1037,6 +1037,8 @@ include::{testDir}/example/ParameterizedTestDemo.java[tags=implicit_conversion_e
 | `double`/`Double`          | `"1.0"`                                  -> `1.0d`
 | `Enum` subclass            | `"SECONDS"`                              -> `TimeUnit.SECONDS`
 | `java.io.File`             | `"/path/to/file"`                        -> `new File("/path/to/file")`
+| `java.lang.Class`          | `"java.lang.Integer"`                    -> `Class.forName("java.lang.Integer")` +
+(remember to use `$` for inner classes, e.g. `"java.lang.Thread$State"`)
```

Comment:
That's not an "inner class". It's a nested type. 😉

Reply:
Please change "inner" to "nested".

Reply:
Let's meet in the middle and say [nested class](https://docs.oracle.com/javase/tutorial/java/javaOO/nested.html).

Reply:
Ummmm.... that's not "in the middle". 😛 

I only requested you change "inner" to "nested" thereby resulting in "nested class". 😉

Reply:
Hah, blessed are those with reading skills. :joy: